pkg/sql/logictest/testdata/logic_test/schema

```diff
@@ -382,3 +382,33 @@ WHERE
 ----
 user1         user1
 user1_schema  user1
+
+# Ensure that we need CREATE on a database to create a schema.
+statement ok
+CREATE DATABASE perms
+
+user testuser
+
+statement ok
+USE perms
+
+statement error pq: user testuser does not have CREATE privilege on database perms
+CREATE SCHEMA test
```

Comment:
Side question: Is there a reason we don't support `CREATE SCHEMA perms.test`?

Reply:
no, i was just sticking to the postgres syntax when implementing this command

Reply:
Got it. Not a big deal but it feels odd/unexpected to me.

Reply:
It's definitely something that could be added